test/unit/enumerable_test.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'set'
+require_relative '../../lib/invoca/utils/enumerable.rb'
+require_relative '../test_helper'
+
+class EnumerableTest < Minitest::Test
+
+  context 'build_hash' do
+    should 'convert arrays of two items to hash key/value pairs' do
```

Comment:
arrays of [key, value] to hash of key => value

Reply:
That reads better!